lib/realtime/server.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Realtime.Server do
+  @moduledoc """
+  Fetches live data from RTR, and forwards it to connected clients.
+  """
+
+  use GenServer
+
+  alias Gtfs.Route
+  require Logger
+
+  defmodule Opts do
+    @type t :: %__MODULE__{
+            url: String.t(),
+            poll_delay: integer
+          }
+    @enforce_keys [
+      :url,
+      :poll_delay
```

Comment:
What about setting a default poll delay, and then this key could be optional? Are there any negatives to that?

Reply:
There's only one non-test place we call this, so a default wouldn't really get us much.